eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/TopologyCheckApp.java

```diff
@@ -45,7 +63,16 @@ public StormTopology execute(Config config, StormEnvironment environment) {
             persistBoltName,
             new TopologyDataPersistBolt(topologyCheckAppConfig),
             topologyCheckAppConfig.dataExtractorConfig.numEntityPersistBolt
-        ).setNumTasks(topologyCheckAppConfig.dataExtractorConfig.numEntityPersistBolt).shuffleGrouping(spoutName);
+        ).setNumTasks(topologyCheckAppConfig.dataExtractorConfig.numEntityPersistBolt).shuffleGrouping(spoutName);        
+       
+        topologyBuilder.setBolt(
+    		parseBoltName,
+    		new HealthCheckParseBolt(),
+    		topologyCheckAppConfig.dataExtractorConfig.numEntityPersistBolt).shuffleGrouping(persistBoltName);      
+        
+        logger.info("start to sink messsage to kafka");
```

Comment:
Storm topology is lazy execution, so in fact, current it's just defining the execution dataflow but not handle data at all, so `start to sink messsage to kafka` is not correct.